stock_picking_show_return/views/stock_picking_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <record id="view_picking_form" model="ir.ui.view">
+            <field name="model">stock.picking</field>
+            <field name="inherit_id" ref="stock.view_picking_form"/>
+            <field name="arch" type="xml">
+                <notebook position="inside">
+                    <page string="Returns" attrs="{'invisible': [('returned_ids', '=', [(6, False, [])])]}">
```

Comment:
Because it's larger and as we are already in the picking form, it will be duplicated.